task_processing/executors/mesos_executor.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import threading
+
+import mesos.native
+from mesos.interface import mesos_pb2
+
+from task_processing.events.event import mesos_status_to_event
+from task_processing.executors.task_executor import TaskExecutor
```

Comment:
in this iteration TaskExecutor sets up the mesos driver and execution framework and wires them together, it holds the driver thread and is basically a gateway to anything platform specific. users instantiating a certain executor don't need to know the underlying interactions, also it would be a bit circular if execution framework owned the driver, because the driver sort of owns the execution framework.